src/static/css/pages/careers.less

```diff
@@ -0,0 +1,3 @@
+.students-and-graduates .media_image {
+    width: 150px
```

Comment:
This should perhaps move to the global media_image pattern overall? Sounds like 150px (and 130px at small sizes) is going to be a pattern across the board.